sheets/api.py

```diff
@@ -570,13 +631,12 @@ def parse_row_and_create_coupons(row_index, row_data):
                 and the data in the sheet do not agree
             SheetCouponCreationException: Raised if there was an error during coupon creation
         """
-        purchase_order_id = row_data[CouponRequestRow.PURCHASE_ORDER_COL_INDEX].strip()
+        coupon_name = row_data[CouponRequestRow.COUPON_NAME_COL_INDEX].strip()
```

Comment:
Should `sheets.factories.CouponGenerationRequestFactory` be updated to generate fake coupon name values?